ChaosMod/Effects/db/Misc/MiscUTurn.cpp

```diff
@@ -36,10 +38,31 @@ static void OnStart()
 	SET_GAMEPLAY_CAM_RELATIVE_HEADING(camHeading);
 }
 
+// effect by Reguas
+
+static void OnStartFake()
+{
+	OnStart();
+
+	GetComponent<EffectDispatcher>()->OverrideEffectNameId("misc_fakeuturn", "misc_uturn");
+
+	WAIT(g_Random.GetRandomInt(6000, 9000));
+
+	OnStart();
+}
+
 // clang-format off
 REGISTER_EFFECT(OnStart, nullptr, nullptr, EffectInfo
 	{
 		.Name = "U-Turn",
 		.Id = "misc_uturn"
 	}
+);
+
+// clang-format off
+REGISTER_EFFECT(OnStartFake, nullptr, nullptr, EffectInfo
+	{
+		.Name = "Fake U-Turn",
+		.Id = "misc_fakeuturn"
```

Comment:
```suggestion
		.Id = "misc_fakeuturn",
		.HideRealNameOnStart = true
```
Otherwise the real name shows up immediately